people.json

```diff
@@ -0,0 +1 @@
+[{"age":21,"name":"Ade","parent_permission":true,"id":509},{"age":13,"name":"Sane","parent_permission":true,"id":191},{"age":20,"name":"Bahati","parent_permission":true,"id":989},{"age":25,"name":"Bahati","parent_permission":true,"id":420},{"age":20,"name":"Bahati","parent_permission":true,"id":609},{"age":24,"name":"Alex","parent_permission":true,"id":308},{"age":45,"name":"RE","parent_permission":true,"id":557}]
```

Comment:
> - [ ] Good job guys👍, Based on the input data we will save, I recommend adding the `"type": teacher or student` field and include `"specialization": ""` for the Teacher class since this information is part of the input data you receive.😊
> 
> Here's an example to illustrate this:
> 
> In your people.json file, the data should be saved in this format:
> 
> ```
> [{"age":21,"name":"Ade","parent_permission":true,"id":509,"type":"student"},{"age":13,"name":"Sane","parent_permission":true,"id":191,"type":"teacher","specialization":"computer"},{"age":20,"name":"Bahati","parent_permission":true,"id":989,"type":"student"},{"age":25,"name":"Bahati","parent_permission":true,"id":420},{"age":20,"name":"Bahati","parent_permission":true,"id":609},{"age":24,"name":"Alex","parent_permission":true,"id":308},{"age":45,"name":"RE","parent_permission":true,"id":557}]
> ```